### DISCUSSION TITLE
posts/rancher-monitoring/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#240] [Comments]
# posts/rancher-monitoring/

Today in this step by step guide, we’ll set up Grafana, Prometheus, and Alertmanager to monitor your Kubernetes cluster. This can be set up really quickly using helm or the Rancher UI. We’ll install and configure, set up some dashboards, and even set up some alerts using Slack. All this and more in this simple to follow, easy tutorial. Setting up Grafana and Prometheus has never been so easy.

https://technotim.live/posts/rancher-monitoring/

**Comment by burninmedia** [comment]
No, notes from your YouTube? Where's the content?

**Reply by timothystewart6** [reply]
Sorry, this was an earlier version, before I started taking notes :)  If you're looking to do this the pure Kubernetes way, see this! https://technotim.live/posts/kube-grafana-prometheus/
